xrpl/asyncio/transaction/main.py

```diff
@@ -335,9 +365,8 @@ async def _calculate_fee_per_transaction_type(
 
     # Multi-signed Transaction
     # 10 drops × (1 + Number of Signatures Provided)
-    if transaction.signers and len(transaction.signers) > 0:
-        base_fee = net_fee * (1 + len(transaction.signers)) + base_fee
-
+    if signers_count is not None and signers_count > 0:
+        base_fee += net_fee * (1 + signers_count)
```

Comment:
This feels like a breaking change - wouldn't we want to check both?

Reply:
I see how this would be a breaking change. However, I am unsure how someone is supposed to use `autofill` for `multisign` with `transaction.signers`. If I knew how, I wouldn't have added in `signers_count`.

For `multisign`, you must make a transaction, then have people sign this, and then submit it. To my knowledge, you must specify `Fee` in the original transaction because people must sign this specific transaction. If you do it this way, `transaction.signers = 0` at the time of creating the transaction, which is bad. So, you must explicitly pass in `signers_count` (which is how xrpl.js uses it).

I am unsure if I am missing something. The original `test_submit_multisigned.py` file doesn't even use autofill, they fill it with `FEE = get_fee(JSON_RPC_CLIENT)`. I'm not sure if logic with `transactions.signers` is tested/used anywhere.

Reply:
I think this is just a bug fix - I don't think it was calculating correctly for multisigning previously.